.github/workflows/TriggerIntegrationTests.sh

```diff
@@ -30,4 +32,4 @@ if [ "$response" == "" ]; then
 else
     echo "Triggering integration tests failed with: '$response'"
     exit 1
```

Comment:
LGTM. Have we looked into issues that could come up due to PR from forked repos?